src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenRefConditionalOperatorTests.cs

```diff
@@ -662,10 +662,7 @@ static void Main()
             comp.VerifyEmitDiagnostics(
                 // (15,27): error CS8168: Cannot return local 'local1' by reference because it is not a ref local
```

Comment:
>because it is not a ref local [](start = 84, length = 29)

because it is?

Reply:
local1 is not a ref local so we cannot return it by reference.   The quality of cascaded error may be improved, but this is how it work right now.

---
In reply to: [139840517](https://github.com/dotnet/roslyn/pull/22203#discussion_r139840517) [](ancestors = 139840517)